Library/Formula/git-hooks.rb

```diff
@@ -0,0 +1,10 @@
+require "formula"
+
+class GitHooks < Formula
+  homepage "https://github.com/icefox/git-hooks"
+  head "https://github.com/icefox/git-hooks.git"
+
+  def install
+    bin.install "git-hooks"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!